ns-gui/src/main/java/de/nocompany/noscrabble/model/punktemultiplikator/punktemultiplikator.java

```diff
@@ -0,0 +1,104 @@
+package de.nocompany.noscrabble.model.punktemultiplikator;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class punktemultiplikator {
+    private static final int NORMAL = 1;
+    private static final int DW = 20; // Doppelter Wortwert
+    private static final int TW = 30; // Dreifacher Wortwert
+    private static final int DL = 2; // Doppelter Buchstabenwert
+    private static final int TL = 3; // Dreifacher Buchstabenwert
+    private static final int[][] punkteMultiplikatoren = {
```

Comment:
Schaut sehr umständlich aus. Sollte perspektivisch in eine ENUM umgewandelt werden.